mlflow/utils/uri.py

```diff
@@ -19,9 +19,16 @@
 _DATABRICKS_UNITY_CATALOG_SCHEME = "databricks-uc"
 
 
-def is_local_uri(uri):
-    """Returns true if this is a local file path (/foo or file:/foo)."""
-    if uri == "databricks":
+def is_local_uri(uri, is_tracking_or_registry_uri=True):
+    """
+    Returns true if the specified URI is a local file path (/foo or file:/foo).
+
+    :param uri: The URI.
+    :param is_tracking_uri: Whether or not the specified URI is an MLflow Tracking or MLflow
+                            Model Registry URI. Examples of other URIs are MLflow artifact URIs,
+                            filesystem paths, etc.
+    """
+    if uri == "databricks" and is_tracking_or_registry_uri:
```

Comment:
If the `uri` isn't an MLflow Tracking or Model Registry URI, then "databricks" is perfectly valid